libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/ContraIndicators.java

```diff
@@ -27,20 +27,32 @@ public Integer getContraIndicatorScore(
     }
 
     public Optional<ContraIndicator> getLatestContraIndicator() {
-        return contraIndicatorsMap.values().stream()
```

Comment:
The existing test for this method was only passing by chance as far as I could tell, relying on the ordering of a map's values. Hence the update.

Reply:
Is this method actually used anywhere?

Reply:
On closer inspection, no, only by the tests. It would be useful if intellij would highlight that situation.\
I'll remove it.